patsylearn/tests/test_patsy_model.py

```diff
@@ -106,6 +107,20 @@ def test_stateful_transform():
     # make sure that mean of training, not test data was removed
     assert_array_equal(data_trans[:, 0], -1)
 
+def test_stateful_transform_dataframe():
```

Comment:
It's actually based on a copy of your test..And I decided to test is as you test, to check the whole flow and not just dataframe output.